Cargo.toml

```diff
@@ -10,19 +10,29 @@ license-file = "LICENSE"
 description = "Library to facilitate DeFi on Liquid"
 
 [dependencies]
+aes-gcm-siv = { version = "0.9", features = ["std"] }
 anyhow = "1"
+async-trait = "0.1"
+bdk = { version = "0.4", default-features = false }
```

Comment:
Do we have dependabot enabled on this repo? Would be good to stay on top of dependency updates!